### DISCUSSION TITLE
Full-Stack Developer, here to help

### DISCUSSION THREAD

**Discussion by amirnazari96** [WeAreMahsaAmini/Core#191] [Introductions]
Hi guys!
Full-Stack developer in different areas, located in Canada! 
I'm here to help, we are more powerful together!

**Comment by ghost** [comment]
https://github.com/WeAreMahsaAmini/Core/discussions/256

**Comment by ehsaaanes** [comment]
Hi
I'm looking for people who can help creating an app for donation. 
I’d appreciate it if you check it here #163
